### ISSUE TITLE
Email notifications hyperlink issue

### ISSUE THREAD

**Issue by lucastbelem** [jesus-collective/mobile#769]
When a DM is sent, user receives email notification. However, when the user clicks on the hyperlink in the email notification, they are taken to the DM page, but it's blank:

<img width="960" alt="1" src="https://user-images.githubusercontent.com/48295718/109999816-a05e8680-7ce0-11eb-92b9-3f8bea4f5f9c.PNG">

**Comment by GeorgeBellTMH** [comment]
I believe this is a testing error...are you logged on as the person that received the email (and haven't deleted that user)?

**Comment by GeorgeBellTMH** [comment]
You should go to the DM page, the user should be listed, but it doesn't select it by default....this is an improvement we need to make to the email - it needs to have the initialUserID=null&initialUserName=null values set.

**Comment by lucastbelem** [comment]
@GeorgeBellTMH no. I sent a DM to Liam in Dev. He received it via email but when he clicked on it, that's what he saw.
Can we at least make it so that when the user lands on the DM page, he/she is able to see all of their DMs on the left? The blank screen feels like an error to the user.

**Comment by GeorgeBellTMH** [comment]
I think the next time you send a message this should be fixed....

**Comment by GeorgeBellTMH** [comment]
Lucas - I'm going to work on this tomorrow...there are a number of issues that need to be solved...

1) Emails for events/groups should take user directly to that event/group (this might be working)
2) Emails for DMs should take user directly to that DM conversation
3) Emails for course mini-screen should take user directly to that mini-screen (may need to have a temporary solution that just ensure you get taken to the course
4) Emails for course assignments should take user directly to the assignment/response screen message (may need a temporary solution that just ensures you get taken to the course)

Can you verify all the text for emails is ok...let me know if we need any changes....

**Comment by lucastbelem** [comment]
@GeorgeBellTMH I'm good with everything you laid out here. 

Some possible changes in bold to the text for emails. 

Jesus Collective Message Alert
You have received a group message on Jesus Collective from Rachel Meredith **(delete "on Jesus Collective")**

I'm going to go for a walk at lunch. **(could we put the content of the message in a container in the email somehow? Don't want to spend too much time on this, but if there is a way to separate the actual message content from the advisory phrase above, that'd be awesome).**

Login **(change to "open on Jesus Collective")**

**Comment by GeorgeBellTMH** [comment]
I've updated the copy....there is a lot more work here than I realized...I'm having to go digging for the data, then have to modify the pages to actually be able to load it...

**Comment by lucastbelem** [comment]
@GeorgeBellTMH thanks for the update. I'll let the team know. I'm wondering if since we can't link directly to the correct spot, if maybe we can link to a different page? Right now it always goes to DMs. Maybe we just link to the dashboard for everything and let the user figure out where they need to go? Is that a bad idea?

**Comment by GeorgeBellTMH** [comment]
I think I can get it to go to the course, maybe even the course details page....working on it...maybe for tonite...

**Comment by lucastbelem** [comment]
Ok that's cool. But when you say, are you saying for all messages received by anyone, it'd take them to the course...or just for course recipients? If we can only send people to one place, I don't think the course is the best spot. But if you're saying this might work to send everyone to DMs but course messages to the course, that'd be a really good compromise.

**Comment by lucastbelem** [comment]
@GeorgeBellTMH I'll hold off messaging the JC team until tomorrow morning then just in case we can get something to work. It's shouldn't be a make or break it, but a really nice to have. 

Thanks man.
